Sources/Build/BuildPlan.swift

```diff
@@ -91,9 +91,9 @@ extension BuildParameters {
         case .auto:
             if configuration == .debug {
                 addIndexStoreArguments = true
-            } else if enableTestDiscovery && target.type == .test {
-                // Test discovery requires an index store for the test targets
-                // to discover the tests
+            } else if target.type == .test {
+                // 👀 not sure
```

Comment:
👀

Reply:
Yes, it is indeed required on a host platform that builds tests.